akka-actor/src/main/scala/akka/io/TcpOutgoingConnection.scala

```diff
@@ -42,7 +42,7 @@ private[io] class TcpOutgoingConnection(_tcp: TcpExt,
       thunk
     } catch {
       case NonFatal(e) ⇒
-        log.debug("Could not establish connection to [{}] due to {}", remoteAddress, e)
+        log.error(e, "Could not establish connection to [{}]", remoteAddress)
```

Comment:
I can understand the `error` level of bind failed, but is this really `error`? How much stack trace garbage will there be in the logs for failing connection attempts?